guides/hosting/installation-updates/deployments/databaseless.md

```diff
@@ -0,0 +1,71 @@
+# Building assets of administration and storefront without a database
+
+It is common to prebuilt assets in professional deployments to deploy the finished assets to the production environment. This task is mostly done by an CI job which doesn't have access to the the production database. Shopware needs access to the database to look up the installed extensions / loading the configured theme variables. To be able to build the assets without a database, we can use static dumped files.
+ This guide requires Shopware 6.4.0 or higher.
+
+
+## Compiling the Administration without database
+
+By default, without a database Shopware considers only the Administration to be built. To include the extensions without a database, you will need to use the `ComposerPluginLoader`. This determines the used plugins by looking up into the project installed dependencies. To get this working the plugin needs to be required in the system using `composer req [package/name]`.
```

Comment:
Maybe rephrase to:

By default, Shopware only builds the Administration without extensions if there is no database connection.